mobile/src/TableHeader.js

```diff
@@ -8,11 +8,13 @@ import client, { Avatar, TitleBar, Color } from '@doubledutch/rn-client'
 export default class TableHeader extends Component {
 
   renderPrompt = (questions) => {
-    const filteredQuestions = Object.values(questions).filter(item => item.block === false) || []
+    var questions = Object.values(questions)
+    if (this.props.currentSort === "My Questions") {questions = questions.filter((item) => item.creator.id === client.currentUser.id)}
+    const filteredQuestions = questions.filter(item => item.block === false) || []
     if (filteredQuestions.length === 0) {
       return (
         <View style={{marginTop: 96}}>
-          <Text style={{marginTop: 30, textAlign: "center", fontSize: 20, color: '#9B9B9B', marginBottom: 5, height: 25}}>Get the Conversation Started!</Text>
+          <Text style={{marginTop: 30, textAlign: "center", fontSize: 20, color: '#9B9B9B', marginBottom: 5, height: 25}}>{(this.props.currentSort === "My Questions") ? "You Haven't Asked a Question Yet" : "Get the Conversation Started!"}</Text>
```

Comment:
This line (and line 12) are a little long and could use breaking up for readability.